src/styles/components/HomeStatic.css

```
@@ -127,7 +127,7 @@
   height: 440px;
   width: 100%;
   background-image: url('../../assets/images/homeStaticSlider.png');
-  background-position: center;
+  background-position: cover;
```

this one also, you mistype background-size with background-position
```suggestion
  background-size: cover;
```

---

You are right, missed prop in css for cover